File: Back/.Net/Modules/Bets/BetFriends.Bets.Infrastructure/Repositories/Sql/DataAccess/AnswerEntity.cs
using BetFriends.Bets.Domain.AnswerBets;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BetFriends.Bets.Infrastructure.Repositories.Sql.DataAccess;

[Table("answers", Schema = "bet")]
public class AnswerEntity
{
    public AnswerEntity()
    {
        
    }
    public AnswerEntity(AnswerBet answerBet)
    {
        var snapshot = answerBet.Snapshot;
        BetId = snapshot.BetId;
        GamblerId = snapshot.MemberId;
        Answer = snapshot.Answer;
    }

    [Column("bet_id"), Required]
    public Guid BetId { get; init; }
    [Column("gambler_id"), Required]
    public Guid GamblerId { get; init; }
    [Column("answer", TypeName = "bit")]
    public bool Answer { get; set; }

    internal void Update(AnswerBetSnapshot snapshot)
    {
        Answer = snapshot.Answer;
    }
}

File: Back/NodeJS/modules/bets/src/infrastructure/DataAccess/entities/AnswerEntity.ts
import { Column, Entity } from "typeorm";
import { AnswerBet } from "../../../domain/answerBets/AnswerBet";

@Entity({
    schema: "bet",
    name: "answers"
})
export class AnswerEntity {
    static Create(answerBet: AnswerBet): AnswerEntity {
        const entity = new AnswerEntity();
        entity.Answer = answerBet.Answer;
        entity.BetId = answerBet.BetId.Value;
        entity.GamblerId = answerBet.GamblerId.Value;
        return entity;
    }

    @Column({
        name: "bet_id"
    })
    BetId: string;
    @Column({
        name: "gambler_id"
    })
    GamblerId: string;
    @Column({
        name: "answer"
    })
    Answer: boolean;
    @Column({
        name: "upserted_date"
    })
    UpsertedDate: Date;
}